spec/system/dashboard/activity_notifications_spec.rb

```diff
@@ -126,15 +127,16 @@
       reply_text = "I'm a reply from the same contributor: #{contributor_two.name}"
       expect(page).to have_text(reply_text)
       find('p', text: reply_text).hover
-      reply_path = conversations_contributor_path(id: contributor_two, reply_to: reply_by_same_contributor.id, anchor: 'chat-form')
+      reply_path = conversations_organization_contributor_path(organization, contributor_two, reply_to: reply_by_same_contributor.id,
```

Comment:
Removed `id: ` named param is fine, I guess.

Reply:
the path helpers are so clever, yes it works..but tbh. i've mixed feelings on the cleverness. I am a bit  puzzled that id:contributor_two seem to have worked..